src/NServiceBus.Transport.Msmq/CheckMachineNameForCompliance.cs

```diff
@@ -25,12 +25,9 @@ public void Check()
             {
                 return;
             }
-
-            Logger.WarnFormat("NetBIOS name [{0}] is longer than 15 characters. Shorten it for DTC to work.", netbiosName);
+            throw new InvalidOperationException($"The NetBIOS name {netbiosName} is longer than 15 characters. Shorten the machine name to 15 characters or less for MSMQ to deliver messages.");
```

Comment:
no value in a `throw new InvalidOperationException`. a `throw new Exception` will suffice